hi-ml-histopathology/src/histopathology/datamodules/panda_module_baseline.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
I don't have strong opinions on which tag we use to define this benchmark/baseline/experiment but I think important we stick to the same all over the repo - here the module and the class are called baseline for example but this name is not used in the container class.